src/core/main.cpp

```diff
@@ -161,6 +161,7 @@ void printHelp()
 		"Actions:\n"
 		"  <no action> [options...] [<project>]  Start LMMS in normal GUI mode\n"
 		"  dump <in>                             Dump XML of compressed file <in>\n"
+		"  compress <in>                         Compress file <n>\n"
```

Comment:
```suggestion
		"  compress <in>                         Compress file <in>\n"
```

Reply:
Merged. It looks like the one who press the merge button on  a suggested change becomes the 'Author'.